dist/libjwt.spec.in

```diff
@@ -0,0 +1,72 @@
+Summary:            JSON Web Token (C implementation)
+Name:               libjwt
+Version:            @VERSION@
+Release:            0%{?dist}
+License:            LGPL v3
+Group:              Development/Tools
+Source:             https://github.com/benmcollins/libjwt/archive/v%{version}.tar.gz
+URL:                https://github.com/benmcollins/libjwt
+BuildRequires:      make libtool automake
+BuildRequires:      gcc doxygen
+Requires:           openssl jansson
+
+
+%description
+JSON Web Tokens are an open, industry standard RFC 7519 method for representing claims
+securely between two parties. This package is a C library implementing JWT functionnality.
+It supports HS256/384/512, RS256/384/512 and ES256/384/512 digest algorithms.
+
+
+%package devel
+Summary:            JSON Web Token (C implementation) development kit
+Group:              Development/Libraries
+Requires:           c-jwt
+
+%description devel
+Development files for the JWT C library.
+
+
+%package devel-docs
+Summary:            JSON Web Token (C implementation) development kit documentation
+Group:              Development/Libraries
+
+%description devel-docs
+Development documentation files for the JWT C library.
+
+%prep
+%autosetup -n %{name}-%{version}
+
+%build
+autoreconf -i
+%configure
+make all doxygen-doc
+
+
+%install
+make install DESTDIR=%{buildroot}
+mkdir -p %{buildroot}%{_datadir}/%{name}
+cp -r doxygen-doc/html %{buildroot}%{_datadir}/%{name}
+
```

Comment:
You've added the check build dependency. Please add a %check stanza to the spec file to run the test suite with "make check"

Reply:
i'm afraid CentOS 7.7 (and RedHat 7 by the way) doesn't provide a version of check capable of linking the executable build by make check. Check version is 0.9.9 and required version is 0.9.10.
Do you still want %check in spec ?

Reply:
Weird. Ubuntu 16.04 has it. Maybe wrap the BuildRequires for check and the %check stanza  in a dist check and only enable it if it's >= 8?

Otherwise, remove check from BuildRequires altogether. No reason to install it if you're not running %check.